.github/workflows/linux_builds.yml

```diff
@@ -59,6 +60,16 @@ jobs:
             # Skip 2GiB artifact speeding up action.
             artifact: false
 
+          - name: Editor with ThreadSanitizer (target=editor, tests=yes, dev_build=yes, use_tsan=yes, use_llvm=yes, linker=lld)
+            cache-name: linux-editor-thread-sanitizer
+            target: editor
+            tests: true
+            sconsflags: dev_build=yes use_tsan=yes use_llvm=yes linker=lld
```

Comment:
Do you need debug symbols for TSAN to do its job properly, or would it work fine without? We could save on cache usage if debug symbols aren't needed.

Reply:
I'd also imagine they're needed for the reports to make any sense.